pymesync/pymesync.py

```diff
@@ -418,11 +420,13 @@ def get_projects(self, query_parameters=None):
             return mock_pymesync.get_projects(slug)
 
         # Attempt to GET projects, then convert the response to a python
-        # dictionary
+        # dictionary. Always returns a list.
         try:
             # Success!
             response = requests.get(url)
-            return self.__response_to_python(response)
+            res_dict = self.__response_to_python(response)
+
+            return [res_dict] if type(res_dict) is dict else res_dict
```

Comment:
Why wouldn't this just be `return [res_dict] if type(res_dict) is not dict else res_dict`?